jwst/pipeline/calwebb_image2.cfg

```diff
@@ -1,5 +1,7 @@
 name = "Image2Pipeline"
 class = "jwst.pipeline.Image2Pipeline"
+output_use_model = True
+suffix = "cal"
```

Comment:
So how does this work when the input is a rateints product and hence the output needs to be calints (not cal)? Also, now that we're adding the resample step to the end of calwebb_image2, the last product generated is an i2d. Where/how does that get specified?